src/content/registry/python.js

```diff
@@ -1,8 +1,99 @@
-export const urlParser = ({ pathname }) => {
-  const [_empty, _part, packageName, packageVersion] = pathname.split('/');
+const registryParser = (pathname) => {
+  const [_empty, _part, name, version] = pathname.split('/');
   return {
-    registry: 'pypi',
-    packageName,
-    packageVersion: packageVersion || undefined,
+    type: 'pypi',
+    name,
+    version: version || undefined,
   };
 };
+
+const docsParser = (pathname) => {
+  const [_empty, name] = pathname.split('/');
+  return {
+    type: 'pypi',
+    name,
+    version: undefined,
+  };
+};
+
+export const urlParser = ({ hostname, pathname }) => {
+  if (hostname.includes('pypi')) return registryParser(pathname);
+  return docsParser(pathname);
+};
+
+const pipOptionsWithArg = [
+  '-c',
+  '--constraint',
+  '-e',
+  '--editable',
+  '-t',
+  '--target',
+  '--platform',
+  '--python-version',
+  '--implementation',
+  '--abi',
+  '--root',
+  '--prefix',
+  '-b',
+  '--build',
+  '--src',
+  '--upgrade-strategy',
+  '--install-option',
+  '--global-option',
+  '--no-binary',
+  '--only-binary',
+  '--progress-bar',
+  '-i',
+  '--index-url',
+  '--extra-index-url',
+  '-f',
+  '--find-links',
+  '--log',
+  '--proxy',
+  '--retires',
+  '--timeout',
+  '--exists-action',
+  '--trusted-host',
+  '--cert',
+  '--client-cert',
+  '--cache-dir',
+];
+const optionWithArgRegex = `( (${pipOptionsWithArg.join('|')})(=| )\\S+)*`;
+const options = /( -[-\w=]+)*/;
+const packageArea = /["']?(?<package_part>(?<package_name>\w[\w.-]*)([=<>~!]=?[\w.,<>]+)?)["']?(?=\s|$)/g;
+const repeatedPackages = `(?<packages>( ${packageArea.source})+)`;
+const whiteSpace = / +/;
+const PIP_COMMAND_REGEX = new RegExp(
+  `(?<command>pip install${optionWithArgRegex}${options.source})${repeatedPackages}`.replaceAll(' ', whiteSpace.source),
+  'g'
+);
+export const parseCommand = (command) => {
```

Comment:
consider rewriting this with a split logic by space

Reply:
Resolved